sync/src/chain.rs

```diff
@@ -82,7 +82,7 @@ const RECEIPTS_PACKET: u8 = 0x10;
 
 const NETWORK_ID: U256 = ONE_U256; //TODO: get this from parent
 
-const CONNECTION_TIMEOUT_SEC: f64 = 30f64;
+const CONNECTION_TIMEOUT_SEC: f64 = 10f64;
```

Comment:
Large timeout and high import speed leads to block queue underrun and sync stalling when waiting for a unresponsive peer for too long.